src/modules/media/elasticsearch/templates/search-query.json

```diff
@@ -2,96 +2,155 @@
       "query": "{{query}}",
+      "type": "phrase",
+      "slop": 5,
+      "boost": 100,
       "fields": [
-        "schema_is_part_of.serie^20",
-        "schema_is_part_of.serie.lemma_split^15",
-        "schema_is_part_of.serie.lemma_split_spell^10",
         "schema_name^20",
-        "schema_name.lemma_split^15",
-        "schema_name.lemma_split_spell^10",
-        "schema_description^15",
-        "schema_description.lemma_split^10",
-        "schema_description.lemma_split_spell^8",
-        "schema_abstract^10",
-        "schema_abstract.lemma_split^8",
-        "schema_abstract.lemma_split_spell^5",
-        "schema_is_part_of.alternatief^10",
-        "schema_is_part_of.alternatief.lemma_split^8",
-        "schema_is_part_of.alternatief.lemma_split_spell^5",
-        "schema_keywords^5",
-        "schema_keywords.lemma_split^2",
-        "schema_keywords.lemma_split_spell^1",
-        "schema_genre^5",
-        "schema_genre.lemma_split^2",
-        "schema_genre.lemma_split_spell^1",
-        "schema_transcript^5",
-        "schema_transcript.lemma_split^2",
-        "schema_transcript.lemma_split_spell^1",
-        "schema_captions^8",
-        "schema_captions.lemma_split^5",
-        "schema_captions.lemma_split_spell^2",
-        "schema_creator.*^20",
-        "schema_contributor.*^20",
-        "schema_spatial_coverage^20",
-        "schema_spatial_coverage.lemma_split^15",
-        "schema_spatial_coverage.lemma_split_spell^10",
-        "schema_temporal_coverage^20",
-        "schema_temporal_coverage.lemma_split^15",
-        "schema_temporal_coverage.lemma_split_spell^10",
-        "dcterms_medium^20",
-        "dcterms_medium.lemma_split^15",
-        "dcterms_medium.lemma_split_spell^10"
-      ],
+        "schema_description",
+        "schema_abstract",
+        "schema_is_part_of.alternatief^20",
+        "schema_is_part_of.archief",
+        "schema_is_part_of.deelarchief",
+        "schema_is_part_of.deelreeks",
+        "schema_is_part_of.programma, ",
+        "schema_is_part_of.reeks",
+        "schema_is_part_of.seizoen",
+        "schema_is_part_of.serie",
+        "schema_keywords",
+        "schema_genre",
+        "schema_transcript",
+        "schema_captions",
+        "schema_creator.*",
+        "schema_contributor.*",
+        "schema_spatial_coverage",
+        "schema_temporal_coverage",
+        "dcterms_medium"
+      ]
+    }
+  },
+  {
+    "multi_match": {
+      "query": "{{query}}",
       "type": "phrase",
-      "operator": "and",
-      "slop": 5
+      "slop": 5,
+      "boost": 80,
+      "fields": [
+        "schema_name.lemma_split^20",
+        "schema_description.lemma_split",
+        "schema_abstract.lemma_split",
+        "schema_is_part_of.alternatief.lemma_split^20",
+        "schema_is_part_of.archief.lemma_split",
+        "schema_is_part_of.deelarchief.lemma_split",
+        "schema_is_part_of.deelreeks.lemma_split",
+        "schema_is_part_of.programma, .lemma_split",
```

Comment:
Ook hier en in volgende schema_is_part_of.programma's de extra komma verwijderen.